tests/Feature/DatabaseSslStatusRefreshTest.php

```
@@ -28,25 +40,66 @@
+        ->not->toHaveKey("echo-private:user.{$this->user->id},DatabaseStatusChanged")
+        ->not->toHaveKey("echo-private:team.{$this->team->id},ServiceChecked")
+        ->not->toHaveKey("echo-private:team.{$this->team->id},ServiceStatusChanged");
+})->with('database-general-forms-without-broadcasts');
 
-it('reloads the mysql database model when refreshing so ssl controls follow the latest status', function () {
+function resolveLivewireListeners(object $component): array
+{
+    // Livewire's HandlesEvents trait declares getListeners() as protected,
+    // so subclasses that override it as public are callable directly, but
+    // subclasses that rely on $listeners are not. Reflection handles both.
+    $method = new ReflectionMethod($component, 'getListeners');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '86', column '19'). (undefined)

(MissingImport)